clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -312,6 +312,8 @@ def warn_drv_yc_multiple_inputs_clang_cl : Warning<
 def warn_drv_potentially_misspelled_joined_argument : Warning<
   "joined argument treated as '%0'; did you mean '%1'?">, InGroup<UnknownArgument>;
 
+def err_drv_too_many_actions: Error<
+    "only one action option is allowed. Got '%0'">;
```

Comment:
You strip a set of `''` later, could you remove them from here instead, or are they part of the substitution syntax?